### PR TITLE
add trivy vulnerability scanning

### PR DESCRIPTION
**Pull request by justyntemme**
Detect out of date/vulnerable packages in the docker container, can fail if it finds issues, but currently just warns.

### CODE REVIEW

**Comment by DavidNix** [comment]
Awesome!

Can you explain the benefits of trivy vs. other vul scanning solutions? 

Additionally, I think this step only runs on `main`, not on every PR. Failing (not warning) is fine if it runs on `main` only.

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -51,4 +51,14 @@ jobs:
           file: Dockerfile
           push: true
           tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
\ No newline at end of file
+          labels: ${{ steps.meta.outputs.labels }}
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
```
Comment (DavidNix) [inline .github/workflows/release.yaml:57]: Can you explain why you chose trivy vs. other vul scanning solutions?

Comment (justyntemme) [reply .github/workflows/release.yaml:57]: Partially experience with bringing pipelines to compliance standards, however from my understanding trivy offers scanning options that others don't use. For example, trivy can also scan a kubernetes environment for misconfigurations, as well as scanning application dependencies, while most simply scan os packages (that's included as well)

Comment (DavidNix) [reply .github/workflows/release.yaml:57]: Awesome! Thanks for the education. I haven't used trivy yet. (My prior experience is with snyk.)

**Review by DavidNix — APPROVED** [review]

**Pull request merged**